ttnn/cpp/ttnn/operations/eltwise/binary_backward/device/binary_backward_op.cpp

```diff
@@ -683,7 +687,7 @@ std::function<std::vector<ttnn::Tensor>(const Tensor&, const Tensor&, const Tens
     }
 }
 
-std::function<std::vector<ttnn::Tensor>(const Tensor&, const Tensor&, const Tensor&, string, const MemoryConfig&)> get_function_type1_w_string(BinaryBackwardOpType OpType){
+std::function<std::vector<ttnn::Tensor>(const Tensor&, const Tensor&, const Tensor&, std::string, const MemoryConfig&)> get_function_type1_w_string(BinaryBackwardOpType OpType){
```

Comment:
Please read my comments in the https://github.com/tenstorrent/tt-metal/pull/10037.
Also why do we pass std::string by value. Probably string_view or at least  const std::string& might be better.

Reply:
100%
This whole file is on fire. I am not touching it in this PR on purpose, but had to do something to unblock w/o switching branches.